changelog/1.20.0.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "1.20.0"
+description: "Released on 06/16/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.20.0.
+
+### Features ✨
+
+- web: A site-wide workspace template policy may now be managed at **Manage >
+  Admin > Templates > Template Policy**. If not set, a sensible default is
+  provided.
```

Comment:
Needs documentation

cc @Emyrk and @scsmithr for review of clog entry

Reply:
Clog entry is good.